### PR TITLE
Dev.publish

### PR DESCRIPTION
**Pull request by roedoejet**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -25,10 +25,6 @@ install:
   - pip3 install -r requirements.dev.txt
   - pip3 install coverage
   - pip3 install codecov
-  - pip3 install gunicorn
```
Comment (dhdaines) [inline .travis.yml:28]: ah! good catch!

File: requirements.min.txt [hunk 2]
```diff
@@ -15,4 +12,3 @@ pympi-ling==1.69
 python-slugify==5.0.0
 soundswallower~=0.4.1
 webvtt-py==0.4.2
-werkzeug==2.0.3
```
Comment (dhdaines) [inline requirements.min.txt:18]: hooray :)

**Review by dhdaines — APPROVED** [review]
Great! This should help our codecov stats as well...

File: requirements.min.txt (hunk 2 above)
Comment (joanise) [reply requirements.min.txt:18]: Don't celebrate too fast, that dependency is still in g2p...

File: .travis.yml (hunk 1 above)
Comment (joanise) [reply .travis.yml:28]: Hey, why do we still have `.travis.yml` at all? We don't use Travis anymore...

**Pull request merged**